### PR TITLE
Add missing translation for the Transfer error

### PR DESCRIPTION
**Pull request by amiskov**
## What issue does this PR close
Closes N/A. See [this message](https://cambiatus.slack.com/archives/CA83HJAAD/p1606953223006400) on Slack.

image

## Changes Proposed ( a list of new changes introduced by this PR)
Add missing translations.

## How to test ( a list of instructions on how to test this PR)
Go to the Transfers page and try to send a large number of coins, like 70000. See the translated error.

### CODE REVIEW

File: public/translations/pt-BR.json [hunk 1]
```diff
@@ -111,7 +111,8 @@
         "This community don't have objectives enabled.": "Esta comunidade não tem objetivos habilitados.",
         "You can't use transfersale to this sale if you aren't part of the community": "Você não pode usar transfersale para esta venda se você não fizer parte da comunidade",
         "Sale creator and sale doesn't match": "O criador da venda e a venda não correspondem",
-        "Invalid number of units": "Número inválido de unidades"
+        "Invalid number of units": "Número inválido de unidades",
+        "overdrawn community limit": "Limite de comunidade a descoberto"
```
Comment (lucca65) [inline public/translations/pt-BR.json:115]:
```suggestion
        "overdrawn community limit": "Saldo insuficiente para realizar essa transferência"
```

File: public/translations/en-US.json [hunk 2]
```diff
@@ -111,7 +111,8 @@
         "This community don't have objectives enabled.": "This community doesn't have objectives enabled",
         "You can't use transfersale to this sale if you aren't part of the community": "You can't use transfersale to this sale if you aren't part of the community",
         "Sale creator and sale doesn't match": "Sale creator and sale doesn't match",
-        "Invalid number of units": "Invalid number of units"
+        "Invalid number of units": "Invalid number of units",
+        "overdrawn community limit": "Overdrawn community limit"
```
Comment (lucca65) [inline public/translations/en-US.json:115]:
```suggestion
        "overdrawn community limit": "Insufficient balance to make this transfer"
```

File: public/translations/es-ES.json [hunk 3]
```diff
@@ -111,7 +111,8 @@
         "This community don't have objectives enabled.": "Esta comunidad no tiene objetivos habilitados.",
         "You can't use transfersale to this sale if you aren't part of the community": "No puede usar la venta transferible para esta venta si no es parte de la comunidad",
         "Sale creator and sale doesn't match": "Creador de venta y venta no coincide",
-        "Invalid number of units": "Número de unidades no válido"
+        "Invalid number of units": "Número de unidades no válido",
+        "overdrawn community limit": "Límite comunitario sobregirado"
```
Comment (lucca65) [inline public/translations/es-ES.json:115]:
```suggestion
        "overdrawn community limit": "Saldo insuficiente para realizar esta transferencia"
```

File: public/translations/cat-CAT.json [hunk 4]
```diff
@@ -111,7 +111,8 @@
         "This community don't have objectives enabled.": "Aquesta comunitat no té objectius activats.",
         "You can't use transfersale to this sale if you aren't part of the community": "No podeu fer servir la venda de transferències a aquesta venda si no formeu part de la comunitat",
         "Sale creator and sale doesn't match": "El creador de la venda i la venda no coincideixen",
-        "Invalid number of units": "Nombre d'unitats no vàlid"
+        "Invalid number of units": "Nombre d'unitats no vàlid",
+        "overdrawn community limit": "Límit de comunitat superat"
```
Comment (lucca65) [inline public/translations/cat-CAT.json:115]:
```suggestion
        "overdrawn community limit": "Saldo insuficient per fer aquesta transferència"
```

**Review by lucca65 — CHANGES_REQUESTED** [review]
The Amharic translation is also missing

`ይህንን ዝውውር ለማድረግ በቂ ያልሆነ ሚዛን`

**Comment by amiskov** [comment]
> The Amharic translation is also missing

@lucca65 Thank you for noticing this and for improving all the errors! I noticed that there's no Amharic translation for contract errors at all. For now I just added one translation related to this PR and added additional item to the #419 issue for adding Amharic translations.

**Review by lucca65 — APPROVED** [review]

**Comment by lucca65** [comment]
I see! It took them a while to translate everything... we have more keys now. Sometimes I wish those json files were elm files so we could type check missing translations 🤔

**Pull request merged**